src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -243,6 +247,21 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Debug,
                 3,
                 "No actions matched the current request. Route values: {RouteValues}");
+
+            _featureNotFound = LoggerMessage.Define<string, string>(
+                LogLevel.Debug,
+                1,
+                "The filter {Filter} could not find the feature {Feature}.");
+
+            _featureIsReadOnly = LoggerMessage.Define<string>(
+                LogLevel.Debug,
+                2,
+                "The implementation of {Feature} is read-only. The attribute cannot be applied.");
+            
+            _maxRequestBodySizeSet = LoggerMessage.Define<string>(
+                LogLevel.Debug,
+                3,
+                "The maximum request body size has been set to {RequestSize}.");
```

Comment:
This one is perfect for the filter that sets a size.

I would use a separate message for the filter that disables the limit and say

> The request body size limit has been disabled.